### ISSUE TITLE
Consider adding a demo or resource for nbgrader assignments

### ISSUE THREAD

**Issue by ixjlyons** [moorepants/resonance#109]
We didn't really explicitly go over how to fill in an assignment notebook. It mostly seems to be going fine, but there are a few quirky things that can be done that should be avoided:

- Duplicating a grading cell. The metadata is copied so there are multiple cells with the same cell-id and validation fails.
- Not using provided "YOUR ANSWER HERE" cells but adding a cell below. It's very clear in the assignment creation interface that there's a cell for the answer to be written but students just see the text so it's not perfectly clear what to do. It can be easy to miss their answer sometimes.
- Using the `notebook` backend. Plots don't show up in the HTML generated for manual grading. Apparently (haven't personally tested) manually running the notebook in the `submitted/` folder and then going back to the grading interface fixes it.

**Comment by ixjlyons** [comment]
Other notes from meeting:

- could add these items to the website
- could also add these notes as a header to each homework (either through nbgrader config or just manually)

**Comment by moorepants** [comment]
@greg1877 FYI, just noticed this old issue that is related to some of the current things you are facing with the student's use of nbgrader notebooks.

**Comment by greg1877** [comment]
Yup!  That is exactly what is going on.  I was going to make another issue,
but its already there.
